subsys/serialization/transport/rp_ll.c

```diff
@@ -0,0 +1,335 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+#include <zephyr.h>
+#include <errno.h>
+#include <drivers/ipm.h>
+
+#include <openamp/open_amp.h>
+#include <metal/sys.h>
+#include <metal/device.h>
+#include <metal/alloc.h>
+
+#include "rp_ll.h"
+
+#include <logging/log.h>
+
+LOG_MODULE_REGISTER(rp_ll_api);
+
+static struct k_sem ipm_event_sem;
+
+/* Stack size of stack area used by rx thread */
+#define STACKSIZE 1024
+
+/* Scheduling priority used by rx thread */
+#define PRIORITY 7
```

Comment:
This must be specified in Kconfig

Reply:
Yes. I did not add it yet, because I consider this file not as a long term solution. See comments at the beginning of file.